tests/Integration/Behaviour/Features/Context/Domain/StoreFeatureContext.php

```diff
@@ -65,15 +139,16 @@ public function bulkToggleStatus(string $action, string $storeReferences): void
         $storeIds = [];
 
         foreach (PrimitiveUtils::castStringArrayIntoArray($storeReferences) as $storeReference) {
```

Comment:
maybe the getShopIdsByReferences() method should be instead of this loop?

Reply:
Indeed, I refactored the ids retrieval. Please tell me if it looks good for you now.

Reply:
looks good :+1: